test/boost/database_test.cc

```diff
@@ -595,10 +595,12 @@ future<> do_with_some_data(std::vector<sstring> cf_names, std::function<future<>
                                                   "not null primary key (p1, c1, c2)",
                                                   cf_name))
                         .get();
-                    f1.get();
-                    e.get_system_keyspace().local().load_built_views().get();
```

Comment:
Unless I'm misunderstanding something, I think it was wrong of you to remove the `f1.get()` - I think you wanted to wait for f1 (for the CREATE MATERIALIZED VIEW to finish building the view) and also for your new f2 (wait for the CREATE INDEX to finish the view building).

Also, I don't understand why you moved the `e.get_system_keyspace().local().load_built_views().get();` line. What is this line even trying to do?

Reply:
right, somehow I messed things up. reverted.
as of `load_built_views`, honestly I dont remember why I added this line back then, it does nothing useful in this context, just getting a list of views which is not used here, thus removed